wpengine/atlas-blueprint-portfolio
# [MERL-159] Add ACM Blueprint export

## Description

This PR adds the Portfolio model from ACM materialised as a zip file from the acm blueprint export command. 


Ticket: https://wpengine.atlassian.net/browse/MERL-159

## How to Test

On a newly created WP Install:

1. Upload the zip file using the media upload. This is due to the fact that currently acm allows imports from a http scheme only.
2. Run the wp acm import command from the command line:
```
❯ wp acm blueprint import http://localhost:10008/wp-content/uploads/2022/03/acm-blueprint.zip
Fetching zip.
Unzipping.
Verifying ACM manifest.
Checking minimum versions.
Importing ACM models and fields.
Importing posts.
Importing post meta.
Deleting zip and manifest.
Success: Import complete
```
3. Navigate to the WP Admin and verify that the model is correct

## Screenshots

<img width="1792" alt="Screenshot 2022-03-02 at 12 49 26" src="https://user-images.githubusercontent.com/328805/156364701-48701a05-5dea-4663-906e-fb771bbb8c1d.png">
